packages/story-editor/src/components/colorPicker/customColorPicker.js

```diff
@@ -116,6 +116,7 @@ function CustomColorPicker({
             onDelete={removeCurrentStop}
             onRotate={rotateClockwise}
             onMove={moveCurrentStopBy}
+            type={type}
```

Comment:
I'd probably have made this a boolean flag, like `canRotate={type === 'linear'}`, but this works too. It's a matter of opinion for sure.